cmd/bot/main.go

```diff
@@ -10,13 +11,18 @@ import (
 )
 
 var (
+	port = flag.Int("p", 8080, "The port used to serve the messenger bot")
```

Comment:
I'd prefer to only use one configuration package.

Could you rewrite this as:

`port = Flag.Int("port", 8080, "The port used to serve the messenger bot")`

You can provide the parameter as `./messenger-bot --port=4040` in the same way you would a normal `flag`.

Remember to remove all references to the `flag` package.